scanners/pshtt.py

```diff
@@ -101,3 +104,7 @@ def scan(domain, options):
     "Base Domain HSTS Preloaded", "Domain Supports HTTPS",
     "Domain Enforces HTTPS", "Domain Uses Strong HSTS", "Unknown Error",
 ]
+
+
+def format_domain(domain):
+    return re.sub("^(https?://)?(www\.)?", "", domain)
```

Comment:
This is in place of the `format_domains` function pshtt uses internally, but which only gets run during CLI use of pshtt. It'd be nice of `pshtt` moved this function deeper inside, so that it got run on everything passed to `inspect_domains`.